### PR TITLE
Log LoadClientId

### PR DESCRIPTION
**Pull request by alfredoihernandez**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
Log LoadClient ID so that test runs may be correlated to log entries
image

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [x] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/552

### CODE REVIEW

File: src/LodeRunner.Core/LodeRunner.Core.xml
Comment (bartr) [inline src/LodeRunner.Core/LodeRunner.Core.xml:391]: is this a generated file?  If so, should it be included in git?

File: src/LodeRunner.Core/Models/LoadClient.cs
Comment (bartr) [inline src/LodeRunner.Core/Models/LoadClient.cs:71]: we should be using the "new ()" syntax - make sure your style cop is up to date

File: src/LodeRunner/src/ValidationTest/Main.cs
Comment (bartr) [inline src/LodeRunner/src/ValidationTest/Main.cs:477]: run code cleanup to remove "this."

File: src/LodeRunner.Core/Models/LoadClient.cs
Comment (bartr) [reply src/LodeRunner.Core/Models/LoadClient.cs:71]: this is being missed because the nuget packages are out of date

Comment (bartr) [reply src/LodeRunner.Core/Models/LoadClient.cs:71]: .editorconfig and stylecop.json are duplicated in the project - it's better to have exactly one of each and reference - you should be able to have each in the src directory

File: src/LodeRunner/src/ValidationTest/Main.cs
Comment (jofultz) [reply src/LodeRunner/src/ValidationTest/Main.cs:477]: Since I've been back from break, I've been adding `this` myself, because stylecop has started erroring for it and with the rule: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1101.md.  So, it might be there, because it got marked.

File: src/LodeRunner.Core/LodeRunner.Core.xml
Comment (jofultz) [reply src/LodeRunner.Core/LodeRunner.Core.xml:391]: Agreed.  Please remove the xml docs files (LodeRunner.Core.xml & LodeRunner.API.xml) from Git and and exclude *.xml in .gitignore.  I don't believe that we have any other XML files we need in source.  Check with @gortegaMS if you're unsure.

File: src/LodeRunner.Core/Models/LoadClient.cs
Comment (jofultz) [reply src/LodeRunner.Core/Models/LoadClient.cs:71]: @bartr, there are several different projects loaded for each solution.  I think there is one of those files for each project, but it means that when you open a solution file you would find them in each project.  We could move it to a single file for each that exists outside the project folder structure where we're storing the .sln files.  Would that be better?

File: src/LodeRunner.Core/LodeRunner.Core.xml
Comment (alfredoihernandez) [reply src/LodeRunner.Core/LodeRunner.Core.xml]: no problem. removed both

File: src/LodeRunner.Core/Models/LoadClient.cs
Comment (alfredoihernandez) [reply src/LodeRunner.Core/Models/LoadClient.cs]: switched syntax

Comment (bartr) [reply src/LodeRunner.Core/Models/LoadClient.cs]: @jofultz  that's where I would suggest putting the .editor and stylecop - that way you have "exactly one" - with the nuget upgrade, I think you're going to want to change some rules, so having one is goodness - the new "new ()" rule is painful to fix (assuming you adopt the rule)

File: src/LodeRunner/src/ValidationTest/Main.cs
Comment (bartr) [reply src/LodeRunner/src/ValidationTest/Main.cs:477]: @jofultz I think this is a case of having an out of date cop file.  When we reviewed the rules almost a year ago, we decided to not use this (thus my comment).  I'm OK either way as long as we're consistent. If you don't use this, you sometimes end up with a few for clarity which is OK.

File: src/LodeRunner.Core/Models/LoadClient.cs
Comment (bartr) [reply src/LodeRunner.Core/Models/LoadClient.cs]: @alfredoihernandez - there are likely hundreds of the new () changes - style cop will catch them once you upgrade and config - team should decide if they are going to adopt the new style or keep the old one then resolve over weeks or a few months if you change. I think the new () syntax is a lot simpler personally - others don't like it. Consistency (and rule setup) are the keys.

Comment (bartr) [reply src/LodeRunner.Core/Models/LoadClient.cs]: @alfredoihernandez - btw - you don't have to fix all of these in this PR - just add an issue(s) and resolve the conversation(s) - don't want to hold up the PR too long

File: src/LodeRunner/src/ValidationTest/Main.cs
Comment (bartr) [reply src/LodeRunner/src/ValidationTest/Main.cs:477]: @jofultz I think VS code clean up will auto fix these if configured correctly

File: src/LodeRunner.Core/Models/LoadClient.cs
Comment (alfredoihernandez) [reply src/LodeRunner.Core/Models/LoadClient.cs]: https://github.com/retaildevcrews/wcnp/issues/556

File: src/LodeRunner/src/ValidationTest/Main.cs
Comment (alfredoihernandez) [reply src/LodeRunner/src/ValidationTest/Main.cs:477]: https://github.com/retaildevcrews/wcnp/issues/556

**Review by jofultz — APPROVED** [review]

**Pull request merged**
